.appveyor.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration for AppVeyor builds
+
+environment:
+  # Appveyor machines come with miniconda already installed.
+  CONDA_INSTALL_LOCN: "C:\\Miniconda-x64"
+  matrix:
+    - TARGET_ARCH: "x64"
+      PYTHON_BUILD_RESTRICTIONS: "3.7*"
+      CONDA_PY: "37"
+
+# We always use a 64-bit machine, but can build x86 distributions
+# with the TARGET_ARCH variable.
+platform:
+    - x64
+
+# See: https://help.appveyor.com/discussions/problems/12705-custom-build-script-script-mode
+build: off
+
+install:
+  # No need to install miniconda because appveyor comes with it.
+  - cmd: SET PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\Scripts;%PATH%
+  - cmd: conda config --set always_yes true
+  - cmd: conda install pip setuptools wheel
+  - cmd: pip install tox
+  - cmd: conda update --quiet conda
+  - cmd: conda install pytorch torchvision cudatoolkit=10.2 -c pytorch
+
+test_script:
+  - "%CMD_IN_ENV% tox"
```

Comment:
can specify the environments `tox -e py,integration` so it doesn't run all of the other tests that aren't really relevant here (like documentation build, flake8, etc.)

Reply:
does `%CMD_IN_ENV%` make sure that the tox is installed on the conda environment?

Reply:
No, I installed it explicitly in the environment under install.
`%CMD_IN_ENV%` just ensures that the command is run within the current environment.